src/EventListener/AddLibrarySubscriber.php

```diff
@@ -53,14 +52,13 @@ public function addLibrary(FilterResponseEvent $event): void
             return;
         }
 
-        // we don't want to add the library code on redirects
-        $statusCode = $event->getResponse()->getStatusCode();
-        if (($statusCode < 200 || $statusCode > 299) && !in_array($statusCode, [404, 500], true)) {
-            return;
-        }
+        $this->tagBag->add(new ScriptTag('var _awev=window._awev||[];', Tags::TAG_EVENT_HANDLER), TagBagInterface::SECTION_HEAD);
```

Comment:
Not sure, but have feeling this should be added after library code inclusion.

Reply:
It's better in the head because then we know if we add scripts further down that the variable `_awev` is set